system/jlib/jarray.cpp

```diff
@@ -46,15 +46,10 @@ void Allocator::_space(size32_t iSize)
     used++;
 }
 
-void Allocator::_ensure(aindex_t next, size32_t iSize)
+void Allocator::_ensure(aindex_t req, size32_t iSize)
 {
-    aindex_t req = used + next;
     if (req > max )
-    {
-        if ((max == 0) && (req < DOUBLE_LIMIT))
-            max = ensurePowerOfTwo(req);
```

Comment:
removed use of ensurePowerOfTwo.
ensurePowerOfTwo definition still exists, but not used.

Reply:
deleted